### ISSUE TITLE
Question: How to restart connector with tasks?

### ISSUE THREAD

**Issue by 0xERR0R** [SourceLabOrg/kafka-connect-client#78]
KafkaConnectClient provides only the `restartConnector(name)` method to restart the connector. Internally used `PostConnectorRestart` class can be parameterized with `withIncludeTasks` and `withOnlyFailed`, but I don't see any possibility to use it directly. How can I restart the connector with all/only failed Tasks?

**Comment by Crim** [comment]
what about adding a method to `KafkaConnectClient` that looks like:

```java
/**
     * Restart a connector
     * https://docs.confluent.io/current/connect/references/restapi.html#post--connectors-(string-name)-restart
     *
     * @param connectorRestartRequest Defines the connector restart request.
     * @return Boolean true if success.
     */
    public Boolean restartConnector(final PostConnectorRestart connectorRestartRequest)
    {
        return submitRequest(connectorRestartRequest);
    }
```

**Comment by Crim** [comment]
I just released version 4.0.2 that includes the above method.  Let me know if this solves this for you or not.

Thanks!

**Comment by 0xERR0R** [comment]
Just tested 4.0.2 and it works like a charm. Thank you! 👍
